src/databricks/labs/ucx/install.py

```diff
@@ -248,21 +248,35 @@ def _create_jobs(self):
         self._create_readme()
         self._create_debug(remote_wheel)
 
+    @staticmethod
+    def _sort_task_list() -> list[Task]:
```

Comment:
```suggestion
    def _sorted_tasks() -> list[Task]:
```

`_sort` implies no retun

Reply:
Does it? Seems arbitrary.